File: find_even_numbers.c
#include <stdio.h>

int main() {
    printf("Even numbers from 1 to 10:\n");
    
    // Loop through numbers 1 to 10
    for (int i = 1; i <= 10; i++) {
        // Check if number is even (divisible by 2)
        if (i % 2 == 0) {
            printf("%d\n", i);
        }
    }
    
    return 0;
}

File: find_even_numbers.py
# Create a list of numbers from 1 to 10
numbers = list(range(1, 11))

# Find even numbers using list comprehension
even_numbers = [num for num in numbers if num % 2 == 0]

# Print the results
print("Even numbers from 1 to 10:")
print(even_numbers)

# Alternative: print each number on a separate line
print("\nEven numbers (one per line):")
for num in even_numbers:
    print(num)